workers/generic-worker/runtime/runtime_freebsd.go

```diff
@@ -0,0 +1,8 @@
+// +build darwin linux freebsd
+
```

Comment:
This new file shouldn't be needed, as `OSUser` struct is already defined in `workers/generic-worker/runtime/runtime_posix.go` and has `freebsd` added as a build constraint.

Reply:
You're right!  There must have been an artifact laying around as I went ahead and did an `rm -r`, rebuilt and got the error.  It's been a while since I've worked with Go forgive my rustiness.